TwitchChatSharp.sln

```diff
@@ -1,12 +1,10 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.168
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TwitchChatSharp", "TwitchChatSharp\TwitchChatSharp.csproj", "{31EED20F-8D47-44D8-A223-7D177CB417F6}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MinimalisticExample", "MinimalisticExample\MinimalisticExample.csproj", "{782B7BDC-0BB0-42EB-8D5F-8DB0F0D4640A}"
```

Comment:
why is this removed?

Reply:
I just don't have any use for it. I'm fairly new to this process, can you just accept the changes related to missing commands? Anyways, just trying to be helpful.